src/ompl/CMakeLists.txt

```diff
@@ -15,10 +15,7 @@ file(GLOB_RECURSE OMPL_SOURCE_CODE datastructures/*.cpp util/*.cpp base/*.cpp ge
 if(NOT OMPL_HAVE_EIGEN3)
     list(REMOVE_ITEM OMPL_SOURCE_CODE
         "${VERSION_DIR}/util/src/ProlateHyperspheroid.cpp"
-        "${VERSION_DIR}/base/samplers/informed/src/PathLengthDirectInfSampler.cpp"
```

Comment:
Yup, I've run make `clean_bindings && make update_bindings` and it seems fine.
This diff is a little misleading, `PathLengthDirectInfSampler.cpp` is still in the list, leaving the only two files that are skipped in the absence of Eigen, `ProlateHyperspheroid.cpp`, which uses Eigen, and `PathLengthDirectInfSampler.cpp` which uses PHS.